flow-typed/platform.js

```diff
@@ -17,6 +17,7 @@ declare type MWPState = {
 		requestLanguage: string,
 		supportedLangs: Array<string>,
 		initialNow: number,
+		isQLd: boolean,
```

Comment:
The acronym-as-past-participle-verb is sort of hard to read here - I'm wondering about just `isQL` (just to eliminate confusion about what the trailing `d` means) or `isImpersonate` - "impersonate" is the more industry-general term, but unfortunately it's almost never used at Meetup so it actually might create more confusion, but "QL" is so... opaque.

Reply:
Agreed, i think anything other than QL would be confusing as its to engrained in our code / culture